editor/src/bundled-dependencies/babelHelpers.ts

```diff
@@ -2,6 +2,7 @@
 // @ts-nocheck
 /* eslint-disable */
 // Generated by running npx babel-external-helpers > babelHelpers.js, renaming, and then adding the above
+// @ts-ignore
```

Comment:
Omg 5 lines of "Please just ignore this damn code!"

Reply:
:D one for humans!